### PR TITLE
Frontend select typescript

### PR DESCRIPTION
**Pull request by thisconnect**
needs frontend-aopp-ui first

### CODE REVIEW

File: frontends/web/src/components/forms/select.tsx [hunk 1]
```diff
@@ -14,26 +15,40 @@
...
-    selected = null,
+    selectedOption,
     ...props
-}) {
+}: TSelectProps) {
     return (
         <div className={select}>
-            {label && <label for={id}>{label}</label>}
-            <select id={id} {...props}>
+            {label && <label for={props.id}>{label}</label>}
```
Comment (benma) [inline frontends/web/src/components/forms/select.tsx:44]: why do `...props` and then `props.id` instead of putting `id` next to the other props above?

**Review by benma — APPROVED** [review]
utACK

File: frontends/web/src/components/forms/select.tsx (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/components/forms/select.tsx:44]: could yes, but then it also needs to be added to the next line, i.e `<select {...props}>` would change to `<select id={id} {...props}>`

Comment (benma) [reply frontends/web/src/components/forms/select.tsx:44]: up to you!

Comment (thisconnect) [reply frontends/web/src/components/forms/select.tsx]: fixed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/3f5ec258e7b3a9aa0cd38e325bf4abe03b2a9f38..360ce5c0aec87ee2b03c40428be86d79a39413a9 also made separate commit and moved select.jsx to tsx first to preserve the history.

**Pull request merged**